include/benchmark/benchmark.h

```diff
@@ -385,27 +385,27 @@ class Counter {
     kDefaults = 0,
```

Comment:
Does the issue also go away if you simply add `U` suffix here instead?

Reply:
Just tried it but, unfortunately, it does not. Nor does casting everything to an `unsigned`. Seems like the compiler is not attempting to change the underlying type - Or doesn't support it?.